monkey/infection_monkey/control.py

```diff
@@ -42,13 +44,24 @@ def wakeup(parent=None, has_internet_access=None):
         if has_internet_access is None:
             has_internet_access = check_internet_access(WormConfiguration.internet_services)
 
+        root = "root"
+        try:
+            if os.getuid():  # won't throw an exception if it's linux
+                root = os.popen('whoami').read()[:-1]  # get the username
+        except AttributeError:
+            if ctypes.windll.shell32.IsUserAnAdmin():
+                root = "Admin"
+            else:
+                root = "NotAdmin"
```

Comment:
Set default to "Unknown standard user" or something similar. Here don't use `:else` and that value will automatically remain as  "Unknown standard user"